src/Workload/README.md

```diff
@@ -18,16 +18,11 @@ The idea, is a project to be able to set `$(UseMaui)`:
 
 * `Microsoft.NET.Sdk.Maui`
 * `Microsoft.Maui.Sdk`
-* `Microsoft.Maui.Resizetizer.Sdk`
-* `Microsoft.Maui.Core.Ref.[platform]`
-* `Microsoft.Maui.Core.Runtime.[platform]`
-* `Microsoft.Maui.Controls.Ref.[platform]`
-* `Microsoft.Maui.Controls.Runtime.[platform]`
-* `Microsoft.Maui.Essentials.Ref.[platform]`
-* `Microsoft.Maui.Essentials.Runtime.[platform]`
+* `Microsoft.Maui.Controls`
+* `Microsoft.Maui.Controls.Compatibility`
 * `Microsoft.Maui.Templates`
```

Comment:
Can you update this doc to describe the layout of the workload vs what is a NuGet?

Reply:
Busy on that, but yeah. Gonna be epic and all that. 😄

Reply:
https://github.com/dotnet/maui/blob/dev/testing-nugets/docs/design/NuGets.md